Podfile

```diff
@@ -28,11 +31,13 @@ end
 # XCT Testing
 
 target :XCTWebDriverAgentLib, :exclusive => true do
+  pod 'fishhook', :git => 'https://github.com/facebook/fishhook.git', :commit => '8dbd09b'
```

Comment:
I'm not sure but I think so because we share the same main.m? was more afraid of breaking it to be honest :P